src/CC-BY-SA-2.0.xml

```diff
@@ -227,30 +227,31 @@
         <item>
             <bullet>7.</bullet>
           Termination
-        <list>
+			<list>
```

Comment:
The cange to this line looks like it's just injecting whitespace.  I'd rather leave that alone (or at least not indent beyond the child tag as you're doing here) until we get #432 or similar.